src/products/products.controller.ts

```diff
@@ -22,6 +23,15 @@ export class ProductsController {
 
   constructor(private readonly productsService: ProductsService) {}
 
+  convertToApi(p: Product): ProductDto {
+    return new ProductDto({
+      name: p.name,
+      category: p.category,
+      photoUrl: p.photoUrl,
+      description: p.description,
+    });
```

Comment:
Also, you can refuse using `convertToApi`at all.

```suggestion
    return new ProductDto(p);
```